api/tests/views_tests.py

```diff
@@ -466,6 +466,23 @@ def test_201_new_user_created_and_202_user_exists(
         assert responses.assert_call_count(self.fxa_verify_path, 2) is True
         assert responses.assert_call_count(FXA_PROFILE_URL, 1) is True
 
+    @responses.activate()
+    def test_failed_profile_fetch_for_new_user_returns_500(self):
+        user_token = "user-123"
+        self._setup_client(user_token)
+        now_time = int(datetime.now().timestamp())
+        exp_time = (now_time + 60 * 60) * 1000
+        _setup_fxa_response(200, {"active": True, "sub": self.uid, "exp": exp_time})
+        # FxA profile server is down
+        responses.add(responses.GET, FXA_PROFILE_URL, status=502, body="")
+
+        # get fxa response with 201 response for new user and profile created
```

Comment:
change (blocking): this is copy-pasta from line 448.
```suggestion
        # assert response with 500 with details
```